src/components/inputs/reactQueryBuilder/PropertyValueEditor.tsx

```diff
@@ -70,8 +71,29 @@ export function PropertyValueEditor(props: ExpertFilterPropertyProps) {
         [valueEditorProps]
     );
 
+    const filter = useMemo(() => {
+        return createFilterOptions<string>();
+    }, []);
```

Comment:
can call createFilterOptions<string>() directly in the below useCallback to avoid unnecessary chaining dependencies